CMakeLists.txt

```diff
@@ -279,7 +296,11 @@ if(${STRINGZILLA_BUILD_TEST})
   if(SZ_PLATFORM_X86)
     # x86 specific backends
     if (MSVC)
-      define_launcher(stringzilla_test_cpp20_x86_serial scripts/test.cpp 20 "AVX")
+      if(CMAKE_SIZEOF_VOID_P EQUAL 8)
+        define_launcher(stringzilla_test_cpp20_x86_serial scripts/test.cpp 20 "SSE2")
+      else()
+        define_launcher(stringzilla_test_cpp20_x86_serial scripts/test.cpp 20 "AVX")
```

Comment:
This doesn’t look right to me. Is it?

Reply:
I can't remember why I did this now, so I'll revert it.
Are you happy to  use AVX for the serial tests?